sass/_jetpack.scss

```diff
@@ -152,5 +160,5 @@ Featured Content
 .featured-container a,
 .featured-container a:hover,
 .featured-container a:visited {
-  color: #fff;
+	color: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.